include/coap/option.h

```diff
@@ -407,4 +409,68 @@ coap_opt_const_value( const coap_opt_t *opt ) {
 
 /** @} */
 
+/**
+ * Representation of chained list of CoAP options to install.
+ *
+ * @code
+ * coap_optlist_t *optlist_head = NULL;
+ * coap_pdu_t *pdu = coap_new_pdu(session);
+ *
+ * ... other set up code ...
+ * coap_insert_optlist(&optlist, coap_new_optlist(COAP_OPTION_OBSERVE,
```

Comment:
The first argument needs to be `optlist_head`, I suppose?

Reply:
Fixed in update.  Have replaced all optlist_head with optlist_chain in header file for clarity